src/components/screens/public/LoginProvider/modules/Provider/Provider.container.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <component
+    :is="currentComponent"
+    :email="email"
+    :error="error"
+    :is-loading="isLoading"
+    :challenge-type="challengeType"
```

Comment:
type and login? Do we need two props? Maybe we should rename. 

-

Reply:
yes, loginChallenge for oauth login (also will rename as you typed) and challengeType - is email|app|sms types for code